core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorManagerServiceSpec.js

```diff
@@ -20,7 +20,7 @@ describe('Simple Editor Manager Service', function() {
   var stubs;
   var data;
   var mockExplorationData = {
-    explorationId: 0,
+    explorationId: '',
```

Comment:
Well, not the empty string -- that is often some kind of sentinel. Any random string of chars, like 'exp_id', would be fine.